automation/check-patch.e2e-bridge-marker-functests.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -xeu
+
+# This script should be able to execute bridge-marker
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-bridge-marker-functests.sh
+
+teardown() {
+    rm -rf "${TMP_COMPONENT_PATH}"
+    cp $(find . -name "*junit*.xml") $ARTIFACTS || true
+    make cluster-down
+}
+
+main() {
+    # Setup CNAO and artifacts temp directory
+    source automation/check-patch.setup.sh
+    cd ${TMP_PROJECT_PATH}
+
+    # Spin-up ephemeral cluster with latest CNAO
+    # this script also exports KUBECONFIG, and fetch $COMPONENT repository
+    COMPONENT="bridge-marker" source automation/components-functests.setup.sh
```

Comment:
We should set `KUBEVIRT_PROVIDER=external`

Reply:
see [comment](https://github.com/kubevirt/bridge-marker/pull/20#discussion_r514249741) in bridge-marker repo PR